HeterogeneousCore/AlpakaInterface/interface/prefixScan.h

```diff
@@ -188,8 +188,17 @@ namespace cms::alpakatools {
         psum[i] = (j < size) ? co[j] : T(0);
       }
       alpaka::syncBlockThreads(acc);
-      blockPrefixScan(acc, psum, psum, blocksPerGrid, ws);
-
+      if (blocksPerGrid <= 1024)
```

Comment:
Here and below, `1024` should be `warpSize * warpSize` ?

Reply:
(the warp size is 64 on AMD MI200/MI300 GPUs, and may be 8 or 16 on Intel integrated GPUs)

Reply:
Indeed!